### PR TITLE
Fix composer and licence

### PR DESCRIPTION
**Pull request by clement-michelet**

### CODE REVIEW

**Comment by pdenis** [comment]
:+1:

**Comment by juchi** [comment]
:+1:

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "phpunit/phpunit": "4.*"
     },
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=5.4",
```
Comment (nicolasThal) [inline composer.json]: Is your bundle compatible with php7?

Comment (clement-michelet) [inline composer.json]: I've added a restriction

**Pull request merged**